pkger/http_remote_service.go

```diff
@@ -157,35 +157,37 @@ func (s *HTTPRemoteService) CreatePkg(ctx context.Context, setters ...CreatePkgS
 // DryRun provides a dry run of the pkg application. The pkg will be marked verified
 // for later calls to Apply. This func will be run on an Apply if it has not been run
 // already.
-func (s *HTTPRemoteService) DryRun(ctx context.Context, orgID, userID influxdb.ID, pkg *Pkg, opts ...ApplyOptFn) (PkgImpactSummary, error) {
-	return s.apply(ctx, orgID, pkg, true, opts...)
+func (s *HTTPRemoteService) DryRun(ctx context.Context, orgID, userID influxdb.ID, opts ...ApplyOptFn) (PkgImpactSummary, error) {
+	return s.apply(ctx, orgID, true, opts...)
```

Comment:
not a blocker for this, but a design consideration to have a think about:

boolean arguments trigger me hard because they provide very little context to what's happening. i assume `true` and `false` passed in to `apply` have to do with dry runs, but that is a wild ass guess that required looking around at the context of these calls to even have an idea. point is, boolean arguments are a kind of hard stop on comprehension and require digging into definitions to understand.

i think it's usually worth considering how the intent of this `true` vs `false` value might be captured in a way that gives context to the reader. making it an enum is one way i can think of that doesn't resort to stringly typed stuff.

Reply:
I hear you, if it were exported, I'd have addressed it. However, it's internal, and don't see it necessary here.